src/TTrigger.cpp

```diff
@@ -1334,5 +1334,5 @@ TTrigger* TTrigger::killTrigger(const QString& name)
     for (auto trigger : *mpMyChildrenList) {
         trigger->killTrigger(name);
     }
-    return 0;
+    return nullptr;
```

Comment:
This actually seems a bit pointless - why does this method bother to always return with _nothing?_